android/app/src/main/java/app/organicmaps/MwmApplication.java

```diff
@@ -142,6 +144,14 @@ public static SharedPreferences prefs(@NonNull Context context)
   public void onCreate()
   {
     super.onCreate();
+
+    //this will enable the strict mode only when the application is in debug mode
+    if (BuildConfig.DEBUG)
+    {
+      enableStrictMode();
```

Comment:
1. Would it be safer to put this check into
```
  static
  {
    System.loadLibrary("organicmaps");
  }
```
?
2. Please follow our coding convention (no bracers for one-liners, space after //, sentences start with a capital letter)

Reply:
Thanks for your suggestion. There would be no harm in putting the debug build check into static like this.
``` java
static
  {
    System.loadLibrary("organicmaps");

    if (BuildConfig.DEBUG)
      enableStrictMode();
  }
```
I will update the it with proper coding convention.